bindings/c/test/mako/mako.cpp

```diff
@@ -1573,8 +1573,16 @@ int Arguments::validate() {
 		logr.error("--num_databases ({}) must be >= number of clusters({})", num_databases, num_fdb_clusters);
 		return -1;
 	}
-	if (num_threads < num_databases) {
-		logr.error("--threads ({}) must be >= number of databases ({})", num_threads, num_databases);
+	if (async_xacts == 0 && num_threads < num_databases) {
```

Comment:
Add comment:
Thread workers are assigned to databases in sync more, so if there are more databases than threads they will be unused